File: javascript/beginner-series-number-3-sum-of-numbers/solution.js
// https://www.codewars.com/kata/55f2b110f61eb01779000053

// Gives the sum of all integers from a to b (inclusive)
function sumRange(a,b) {
  const smaller = a <= b ? a : b;
  const larger = a + b - smaller;
  
  var sum = 0;
  for (var i = smaller; i <= larger; i++) {
    sum += i;
  }
  return sum;
}

// function name as required by tests
var GetSum = sumRange;

File: python/beginner-series-number-3-sum-of-numbers/solution.py
# https://www.codewars.com/kata/55f2b110f61eb01779000053

def get_sum(a,b):
    if a == b:
        return a
    
    larger  = a
    smaller = b
    if a < b:
        smaller = a
        larger  = b
    
    sum = larger
    for i in range(smaller, larger):
        sum += i
    
    return sum

